_pages/home/index.md

```diff
@@ -1,13 +1,133 @@
 ---
 title: Home
-access:
-  public: true
+hero:
+  title: |
+    Build your career,
+    shape the future.
+  image: /images/home/hero-fpo.jpg
+  image_2x: /images/home/hero-fpo@2x.jpg
+  enabled: true
+sidebar: []
 lifecycle:
   expires_at: ""
-updated_at: 2022-04-26T00:00:00.000Z
+access:
+  public: true
+bottom_content:
+  - block: "home-testimonials/block"
+  - block: "home-blog-headlines/block"
+  - block: "home-wellness/block"
```

Comment:
[issue] I'm not seeing these blocks in the home page netlify editor.

There's a mix of `bottom_content` and `bottom_blocks` between this, netlify_config, and the Page model and view. I think it would be clearer if all three places used the same name.